hw/ip/lc_ctrl/rtl/lc_ctrl_state_pkg.sv.tpl

```diff
@@ -136,11 +136,13 @@ package lc_ctrl_state_pkg;
   // Derived enum types //
   ////////////////////////
 
-  typedef enum logic [LcStateWidth-1:0] {
+  typedef logic [LcStateWidth-1:0] lc_state_t;
+  typedef enum lc_state_t {
```

Comment:
Thank you, the solution LGTM!
Could you add some comments to specify lc_state_t is used for interface to avoid xcelium errors?

Reply:
@cindychip I've added some comments.